dhctl/pkg/kubernetes/actions/deckhouse/pod.go

```diff
@@ -31,7 +31,7 @@ func GetPod(kubeCl *client.KubernetesClient) (*v1.Pod, error) {
 	}
 
 	if len(pods.Items) != 1 {
-		return nil, ErrListPods
+		return nil, ErrMultiplePodsFound
```

Comment:
This is not true. The error will also be shown if there is no Deckhouse pod in the cluster (according to the clause).